clients/erlang/include/space.hrl

```diff
@@ -0,0 +1,25 @@
+-record(config, {bots         :: integer(),
+                 field_radius :: integer(),
+                 move         :: integer(),
+                 start_hp     :: integer(),
+                 cannon       :: integer(),
+                 radar        :: integer(),
+                 see          :: integer(),
+                 max_count    :: integer(),
+                 asteroids    :: integer(),
+                 loop_time    :: integer(),
+                 no_wait      :: boolean()}).
+
+-record(bot, {bot_id  :: integer(),
+              name    :: string(),
+              team_id :: integer(),
+              hp      :: integer(),
+              alive   :: boolean(),
+              pos     :: {integer(), integer()}}).
+
+-record(team, {name    :: string(),
+               team_id :: integer(),
+               bots    :: [#bot{}]}).
+
+%% To be changed by AI implementer:
+-record(state, {some_data :: string()}).
```

Comment:
Yes, not really happy about that. What we could do is send 'no_state' as State to the ai module the first time, pattern match for that, let the bot initialize its own local state record, and then pass that on to the ai logic function that expects an initialized state